hack/local-up-cluster.sh

```diff
@@ -647,7 +648,7 @@ function start_cloud_controller_manager {
     fi
 
     CLOUD_CTLRMGR_LOG=${LOG_DIR}/cloud-controller-manager.log
-    ${CONTROLPLANE_SUDO} "${GO_OUT}/hyperkube" cloud-controller-manager \
+    ${EXTERNAL_CLOUD_PROVIDER_BINARY:-${GO_OUT}/hyperkube cloud-controller-manager} \
```

Comment:
We lost sudo. Am happy if it still works. Every sudo that is gone is a good sudo.

Reply:
hmmm, we'll need it for openstack to get to config drive. will add it back in a few

Reply:
concerning that you need root access for a controller, isn't it?

Reply:
yep. will have to figure out a way to remove that dependency (for config drives)